config/crd/bases/anywhere.eks.amazonaws.com_clusters.yaml

```diff
@@ -277,6 +277,30 @@ spec:
           status:
             description: ClusterStatus defines the observed state of Cluster
             properties:
+              eksdReleaseRef:
+                description: EksdReleaseRef defines the properties of the EKS-D object
```

Comment:
Yaml is case sensitive so you probably want to use the _actual_ key in the description: `eksdReleaseRef`. Ditto for the other keys.